.gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
 node_modules
 npm-debug*.log*
+index.html
```

Comment:
I'm concerned this will prevent the built index.html files from being committed when the Travis script publishes the `gh-pages` branch.

Reply:
FYI, you can add private ignore entries to `.git/info/exclude` in your local copy, that way you can keep the status clean without having to modify this tracked file.

Reply:
Ooh! That's a great tip! Thank you!